SU-HSDO/suhumsci
# STN-231: IE 11 Navigation Bugs

# READY FOR REVIEW

## Summary
This pull request does 2 things:
- Refactors the secondary nav to use our custom JS and CSS
- Fixes any console errors in IE 11 (this is also the reason why we removed Decanter JS for secondary nav, it was causing errors)

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Run `npm test`
2. Run `npm start` to build new assets
3. Review the main nav and secondary nav in a modern browser to ensure everything still works as expected at all screen sizes.
4. Review the main and secondary nav in IE 11 and ensure that it now behaves like modern browsers do.
5. Checkout the Courses page and ensure that the JS is not breaking the main navigation on the page.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
